BookClub/views/forum/forum.py

```diff
@@ -206,10 +206,11 @@ def test_func(self):
             return False
 
     def get_success_url(self):
-        self.kwargs.pop('post_id')
-        if self.kwargs.get('club_url_name') is not None:
-            return reverse('club_forum', kwargs=self.kwargs)
+        forum_club = ForumPost.objects.get(pk=self.kwargs['post_id']).forum.associated_with
```

Comment:
forum__associated_with is better for database access.
Why is post_id not popped in this case?

Reply:
post_id not popped cuz the kwargs aren't used at all in the "then" branch, so, just avoiding an unnecessary code of line.

So, replace "forum.associated_with" with "forum__associated_with"?

Reply:
yes, it should be more efficient as it doesn't look up the entire forum object

I see there's no need for post id then. Makes sense.